api/src/main/java/org/openmrs/module/patientmatching/PatientMatchingActivator.java

```diff
@@ -226,8 +226,7 @@ public void willStop() {
                 try {
                     Context.getSchedulerService().shutdownTask(td);
                 } catch (SchedulerException e) {
-                    // TODO Auto-generated catch block
-                    e.printStackTrace();
+                    log.error("SchedulerException caught while shutting down Patient Matching Module ...", e);

```

Comment:
Could you improve this message to something like "SchedulerException encountered ..."?